apps/advisor/pkg/app/checkscheduler/checkscheduler_test.go

```diff
@@ -11,11 +11,12 @@ import (
 	"github.com/grafana/grafana-app-sdk/resource"
 	advisorv0alpha1 "github.com/grafana/grafana/apps/advisor/pkg/apis/advisor/v0alpha1"
 	"github.com/grafana/grafana/apps/advisor/pkg/app/checks"
+	"github.com/grafana/grafana/pkg/infra/log"
 	"github.com/stretchr/testify/assert"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func TestRunner_Run_ErrorOnList(t *testing.T) {
+func TestRunner_Run_NoErrorOnList(t *testing.T) {
```

Comment:
Sorry I missed this from before but should this test name be updated (or maybe include a `t.Run` to give more info) since we have listFunc still returning an error?